requirements/edx/github.txt

```diff
@@ -34,7 +34,7 @@ git+https://github.com/edx/rfc6266.git@v0.0.5-edx#egg=rfc6266==0.0.5-edx
 -e git+https://github.com/gabrielfalcao/lettuce.git@b18b8fb711eb7a178c58574716032ad8de525912#egg=lettuce=1.8-support
 
 # Our libraries:
--e git+https://github.com/edx/XBlock.git@32fca2a954745315be97b91ef0d5ad4eb38cf365#egg=XBlock
+-e git+https://github.com/edx/XBlock.git@0c1f796cb63dae8363b5530a432a19e84d22983f#egg=XBlock
```

Comment:
https://github.com/edx/XBlock/commit/a20c70f2e3df1cb716b9c7a25fecf57020543b7f is the tip of master — should we do that one?